camelConnector/src/main/resources/artifactMap.json

```diff
@@ -1,5 +1,5 @@
 {
-    "camelVersion": "3.18.3",
```

Comment:
side effect of version update.  But we don't really want to run this for each build, so we check it in...